### ISSUE TITLE
[tvOS] Unable to Choose Alternate Movie Version (e.g., Color, BW)

### ISSUE THREAD

**Issue by isaacsando** [jellyfin/Swiftfin#1320]
### Describe the bug

I'm unable to choose an alternate version of a movie on the Apple TV. A drop-down exists on the 1.3 iOS version (iPhone 14), however, no drop-down exists on the tvOS version.

My Jellyfin folder for the movie looks like this:
```
  Movies
    Example Movie (2024)
      Example Movie (2024) - BW.mp4
      Example Movie (2024) - Color.mp4
```

I also have the Video Player Type set to Swiftfin.

I built Swiftfin from commit 128381a4 and installed it locally using Xcode.

### Application version

1.3

### Where did you install the app from?

None

### Device information

Apple TV 4K

### OS version

tvOS 18.1 (22J580)

### Jellyfin server version

10.10.1

**Comment by warnerbryce** [comment]
Probably linked with this issue 
https://github.com/jellyfin/Swiftfin/issues/1096

Did the version picker works for you on the iOS version ?

**Comment by isaacsando** [comment]
No, I should've double-checked that. The version picker shows the BW and Color versions, but if I choose Color it doesn't change to that. Choosing a version does not have any effect.
